openfe/tests/setup/test_network_planning.py

```diff
@@ -130,21 +130,7 @@ def test_radial_network_index_error(toluene_vs_others, lomap_old_mapper):
             ligands=ligands, central_ligand=2077,
             mappers=lomap_old_mapper, scorer=None,
         )
-
-
-def test_radial_network_self_central(toluene_vs_others, lomap_old_mapper):
```

Comment:
This can not happen anymore with the current Konnektor and OpenFE implementation.
I propose removing the test.

Reply:
By "this cannot happen" do we mean that Konnektor just gracefully handles things such that the network cannot end up with a self edge?

If so, then my suggestion is to keep the code that throws a warning?

If it's more "if you try to do that Konnektor will error", then we should retain the behaviour that pops the ligand out and throws the warning, with a deprecation warning that this behaviour will go away in 2.0.